src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidator.java

```diff
@@ -166,6 +170,26 @@ public void validateOccupationSection(HttpServletRequest request, List<ApiError>
         validateOccupation(request, errorList, dto);
     }
 
+    public void validateCorrespondenceAddressSection(HttpServletRequest request, List<ApiError> errorList, OfficerFilingDto dto, AppointmentFullRecordAPI appointment) {
+        // If hasBeenUpdated boolean is true or null then continue
```

Comment:
should only be for true that we validate?

Reply:
I don't think so? A software filer will not send the hasBeenUpdated boolean but we still want to validate

Reply:
Yeah, you are right.